oci/oci-how-to/create-chiselled-ubuntu-image.rst

```diff
@@ -56,18 +56,18 @@ Start by creating a Dockerfile with the following content:
    FROM scratch
    COPY --from=builder /rootfs /
 
-Build the chiselled Ubuntu base image by running:
+Build the chiselled Ubuntu image by running:
 
 ..  code-block:: bash
    
-   docker build -t chiselled-ubuntu-base:latest . --build-arg ARCH=<your_arch> # example ARCH=amd64
+   docker build -t chiselled-ubuntu:latest . --build-arg ARCH=<your_arch> # example ARCH=amd64
 
 You'll then find yourself with a new container image with approximately 5MB (or 2.5MB when compressed). 
 
 Build the application image
 ---------------------------
 
-Now that you have the ``chiselled-ubuntu-base:latest`` image, you can simply add your compiled application to the image and run it from there. For the sake of simplicity, this guide will give your three very simple “Hello World” application examples for C, Go and Rust.
+Now that you have the ``chiselled-ubuntu:latest`` image, you can simply add your compiled application to the image and run it from there. For the sake of simplicity, this guide will give you three very simple “Hello World” application examples for C, Go and Rust.
```

Comment:
Do we need to specify the location of the source code (app.c)? (So that the ``COPY app /`` in the new Dockerfile works correctly?

Reply:
There's no need because the command below will generate the `app` file in the current directory. So as long as all commands are run in the same directory (no matter what that is), it will work